library/src/main/java/com/otaliastudios/zoom/ZoomApi.kt

```diff
@@ -200,13 +222,16 @@ interface ZoomApi {
     fun setTransformation(@Transformation transformation: Int, gravity: Int)
 
     /**
-     * Sets the policy to use when content is smaller than the container.
-     * Defaults to [SMALLER_POLICY_CENTER]
-     * which means that the content will be centered.
+     * Sets the content alignment. Can be any of the constants defined in [com.otaliastudios.zoom.Alignment].
+     * The content will be aligned and forced to the specified side of the container.
+     * Defaults to [com.otaliastudios.zoom.Alignment.CENTER].
      *
-     * @param policy the policy
+     * Of course, this is disabled when the content is larger than the container,
```

Comment:
> Of course, [...]
I don't like this phrasing... I would go with:
>  Keep in mind that this is disabled when the content is larger than the container, because a forced alignment at all times would result in part of the content being unreachable.
But do as you like.